### ISSUE TITLE
Auto picks EA "persona" 

### ISSUE THREAD

**Issue by Dumbbelly** [hunterjm/futgui#125]
Problem being when an EA account is linked to multiple gamertags/PSN's etc... it will not let you select which if they are both on the same platform - and despite owning fifa 17 I only have access to transfer market on one account ... unfortunately not the one it selects. EA are no use on the subject unfortunately...

**Part issue / part request... the issue being no current way to un link gamertags and therefore being stuck using account#2 - and request.. a way of selecting which gamertag/persona you wish to use...

I'm sure you'll be much better on this than EA anyway - Your work on this is greatly appreciated by me & the 14 year old brother...

**Comment by hunterjm** [comment]
You are correct that it auto selects a persons to use. I'll see what I can do to add the option to pick a persona.
